### ISSUE TITLE
Migration to STIX 2.1

### ISSUE THREAD

**Issue by SamuelHassine** [mitre/cti#79]
Hello,

We would like to have these data using the STIX 2.1 standard.

Kind regards,
Samuel

**Comment by isaisabel** [comment]
Hi @SamuelHassine,

Sorry for the long delay on this, I just wanted to post an update saying that we will be upgrading to STIX 2.1 alongside the release of the [ATT&CK Workbench](https://youtu.be/XpHMTZbuboc?t=4388) project later this year. See also #66 for some more information about new features of the data model which will be accompanying that upgrade.

We will likely maintain a STIX 2.0 version of ATT&CK through ATT&CK versions 9 and 10 to give people time to upgrade to the new representation (though I don't expect there will be many breaking changes).

**Comment by Radu3000** [comment]
@isaisabel ,

Can you please shed some light about "upgrading to STIX 2.1 alongside the release of the ATT&CK Workbench project"?

Does it mean that other types (like Infrastructure) could be part of Attack?

Regards,
Radu

**Comment by isaisabel** [comment]
No, I mean that the ATT&CK Workbench project will be implemented to use STIX 2.1, not STIX 2.0. The actual set of SDO types used in the dataset won't be changing significantly (the only major additions being those for #130 and #66).

Subsequent releases (ATT&CK versions 10 and on most likely) will be using STIX 2.1, though to help users upgrade we'll also maintain a STIX 2.0 dataset for some period of time.No, I mean that the ATT&CK Workbench project will be implemented to use STIX 2.1, not STIX 2.0. The actual set of SDO types used in the dataset won't be changing significantly (the only major additions being those for #130 and #66).

Subsequent releases (ATT&CK versions 10 and on most likely) will be using STIX 2.1, though to help users upgrade we'll also maintain a STIX 2.0 dataset for some period of time.

**Comment by isaisabel** [comment]
ATT&CK in STIX 2.1 is now available on our `attack-stix-data` repository. 

https://github.com/mitre-attack/attack-stix-data
